app/views/posts/show.html.erb

```diff
@@ -12,8 +12,8 @@
           <div class=" dropdown">
             <a class="test" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true"></a>
             <ul class="dropdown-menu box">
-              <li><%= link_to 'Edit', edit_post_path(@post), { method: :get, :class => 'dropdown-item'} %></li>
-              <li><%= link_to 'Delete', destroy_post_post_path(@post), method: :patch, data: { confirm: 'Are you sure?' }, :class => "dropdown-item" %></li>
+              <li><%= link_to 'Edit', edit_post_path(@post), { method: :get, class: 'dropdown-item'} %></li>
+              <li><%= link_to 'Delete', post_path(@post), method: :patch, data: { confirm: 'Are you sure?' }, class: "dropdown-item" %></li>

```

Comment:
Here method should be `:delete` instead of `:patch`. patch is used for updating the record but for such links where we know that it will delete the record we use delete method

Reply:
updated